doc/api/packages.md

```diff
@@ -323,6 +315,16 @@ package. It is not a strong encapsulation since a direct require of any
 absolute subpath of the package such as
 `require('/path/to/node_modules/pkg/subpath.js')` will still load `subpath.js`.
 
+For packages supporting Node.js < 12.7.0 it is necessary to include the `"main"`
```

Comment:
More than old versions of Node.js, there are tools (e.g. bundlers) that don't support `"exports"`, maybe we should just say "to maximize backward-compatibility"?

Reply:
Yeah I can definitely call out other integration workflows further.

Reply:
I've clarified this section, let me know if that works better.